Translation corrections (#115)
07th-mod/higurashi-console-arcs: Update/some_010.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300521", 256, TRUE);
	OutputLine(NULL, "「——と、いうわけだよ。",
			NULL, "\"...And there you have it.", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300522", 256, TRUE);
	OutputLine(NULL, "だからどーんと行って来い、夏美！」",

> 			NULL, " So head out and knock this out of the park, Natsumi!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " So head out there and knock this out of the park, Natsumi!\"", Line_Normal);
```

//r授業が終わるなり、得意満面にばんっ！　と私の机の上にチケットを置いた（叩きつけた？）千紗登ちゃんの笑顔に押されるかたちで、とりあえずその１枚を手に取る。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "授業が終わるなり、得意満面にばんっ！　と私の机の上にチケットを置いた（叩きつけた？）千紗登ちゃんの笑顔に押されるかたちで、とりあえずその１枚を手に取る。",
			NULL, "As soon as class ended, Chisato-san placed (slammed?) the tickets on my desk with a triumphant look on her face. Overwhelmed by her smile, I took one of them to be safe.", Line_Normal);
	ClearMessage();

//r日にちは今週の日曜日までで、金額は２５００円と少し高め。どうやら普通の席よりも、かなりいい場所で観れるようだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "日にちは今週の日曜日までで、金額は２５００円と少し高め。どうやら普通の席よりも、かなりいい場所で観れるようだ。",

> 			NULL, "The date was this Sunday. The price iwa 2,500 yen, a little expensive. Apparently we had better seats than average.", Line_Normal);

Reviewer: ```suggestion
			NULL, "The date was this Sunday. The price was 2,500 yen, a little expensive. Apparently we had better seats than average.", Line_Normal);
```

//r鼻先数ミリのところで思いっきり吼えられて、私は後ろにのけぞる。…椅子から転げ落ちなかっただけでも、たぶん相当な幸運だった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "鼻先数ミリのところで思いっきり吼えられて、私は後ろにのけぞる。…椅子から転げ落ちなかっただけでも、たぶん相当な幸運だった。",

> 			NULL, "She shouted right in my face, making me throw my head back. ...By pure dumb luck, I managed to avoid falling out of my chair.", Line_Normal);

Reviewer: ```suggestion
			NULL, "She shouted right in my face, causing me to flinch backwards. ...By pure dumb luck, I managed to avoid falling out of my chair.", Line_Normal);
```

			NULL, "\"Look here, Natsumi!", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300530", 256, TRUE);
	OutputLine(NULL, "　こうやって映画館の特別席チケットを１枚ペアで渡したら、普通想像することっていったらひとつしかないでしょ？」",

> 			NULL, " One pair of tickets for special seats at the movie theater? Shouldn't that bring one particular something to mind?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " One pair of tickets for special seats at the movie theater? Shouldn't that bring something in particular to mind?\"", Line_Normal);
```

	ClearMessage();

//夏美rvS04/36/310100866.「ひとつって、kvS04/36/310100867.…だから、一緒に映画に」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#94a567>夏美</color>", NULL, "<color=#94a567>Natsumi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 36, "ps3/s04/36/310100866", 256, TRUE);
	OutputLine(NULL, "「ひとつって、",

> 			NULL, "\"One particular something,", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Something in particular,", Line_WaitForInput);
```

//rまたしても怒声を浴びて、私は思わずそばに立つもう一人の親友の背後に隠れる。私たちのやり取りを間近で呆れながら見ていた珠子ちゃんは、何やってんだかと深いため息とともに立ちはだかって言った。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "またしても怒声を浴びて、私は思わずそばに立つもう一人の親友の背後に隠れる。私たちのやり取りを間近で呆れながら見ていた珠子ちゃんは、何やってんだかと深いため息とともに立ちはだかって言った。",

> 			NULL, "Faced with yet another shoute, I hid behind my other best friend who was standing next to me. Tamako-chan, who'd been watching our interaction up close with exasperation, let out a deep sigh as she stood between Chisato-chan and I, then began to speak.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Faced with yet another shout, I hid behind my other best friend who was standing next to me. Tamako-chan, who'd been watching our interaction up close with exasperation, let out a deep sigh as she stood between Chisato-chan and me, then began to speak.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6f6599>珠子</color>", NULL, "<color=#6f6599>Tamako</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 38, "ps3/s04/38/310400207", 256, TRUE);
	OutputLine(NULL, "「あのね、千紗登。",

> 			NULL, "\"Listen here, Chisato?", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Listen here, Chisato.", Line_WaitForInput);
```
